projects/libyaml/build.sh

```diff
@@ -18,9 +18,22 @@
 ./configure
 make "-j$(nproc)"
 
-$CXX $CXXFLAGS -std=c++11 -Iinclude \
-    $SRC/libyaml_fuzzer.cc -o $OUT/libyaml_fuzzer \
-    $LIB_FUZZING_ENGINE src/.libs/libyaml.a
+for fuzzer in $SRC/*_fuzzer.c; do
+  fuzzer_basename=$(basename -s .cc $fuzzer)
 
-cp $SRC/libyaml_fuzzer_seed_corpus.zip $OUT/
-cp $SRC/*.dict $SRC/*.options $OUT/
+  $CC $CFLAGS \
+      -Iinclude -c $fuzzer \
+      -o $fuzzer_basename.o
```

Comment:
Is it necessary to compile these files with `$CC` before linking with `$CXX`? I wouldn't think so, but you're the one who's actually tried it.

Reply:
I'm running into the following error without using `$CC` first:
```
/usr/local/lib/clang/11.0.0/lib/linux/libclang_rt.fuzzer-x86_64.a(fuzzer.o): In function `main':
/src/llvm-project/compiler-rt/lib/fuzzer/FuzzerMain.cpp:19: undefined reference to `LLVMFuzzerTestOneInput'
clang-11: error: linker command failed with exit code 1 (use -v to see invocation)
Building fuzzers failed.
```
This seems like a common issue, as I've already encountered a couple of [other projects](https://github.com/google/oss-fuzz/blob/master/projects/binutils/build.sh#L41) using the same format. Using the `-c` flag alone gives a bunch of warnings about treating C++ input as pure C, so I think we should continue to compile using `$CC` first.